bridge/server/app.ts

```diff
@@ -263,32 +267,32 @@ function setupLookAndFeel(url: string): void {
             zip.extractAllToAsync(destDir, true, false, (error?: Error) => {
               unlinkSync(destFile);
               if (error) {
-                console.error(`[ERROR] Error while extracting custom Look-and-Feel file. ${error}`);
+                log.error(`[ERROR] Error while extracting custom Look-and-Feel file. ${error}`);
                 return;
               }
-              console.log('Custom Look-and-Feel downloaded and extracted successfully');
+              log.info('Custom Look-and-Feel downloaded and extracted successfully');
             });
           } catch (error) {
-            console.error(`[ERROR] Error while extracting custom Look-and-Feel file. ${error}`);
+            log.error(`[ERROR] Error while extracting custom Look-and-Feel file. ${error}`);
           }
         });
         file.on('error', async (err) => {
           file.end();
           try {
             await unlink(destFile);
           } catch (error) {
-            console.error(`[ERROR] Error while saving custom Look-and-Feel file. ${error}`);
+            log.error(`[ERROR] Error while saving custom Look-and-Feel file. ${error}`);
```

Comment:
What would be nice is to have a artificial ID for requests. So you can group logs and trace a workflow...